app/ui/package.json

```diff
@@ -25,57 +24,62 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "4.4.5",
-    "@angular/common": "4.4.5",
-    "@angular/compiler": "4.4.5",
-    "@angular/core": "4.4.5",
-    "@angular/forms": "4.4.5",
-    "@angular/http": "4.4.5",
-    "@angular/platform-browser": "4.4.5",
-    "@angular/platform-browser-dynamic": "4.4.5",
-    "@angular/router": "4.4.5",
-    "@atlasmap/atlasmap.data.mapper": "^1.32.0-SNAPSHOT.20180110153059",
-    "@ng-dynamic-forms/core": "1.4.31",
-    "@ng-dynamic-forms/ui-bootstrap": "1.4.31",
+    "rxjs": "^5.5.2",
+    "zone.js": "^0.8.14",
+    "core-js": "2.4.1",
+    "@angular/animations": "^5.2.0",
+    "@angular/cli": "^1.6.5",
+    "@angular/common": "^5.2.0",
+    "@angular/compiler": "^5.2.0",
+    "@angular/core": "^5.2.0",
+    "@angular/forms": "^5.2.0",
+    "@angular/http": "^5.2.0",
+    "@angular/platform-browser": "^5.2.0",
+    "@angular/platform-browser-dynamic": "^5.2.0",
+    "@angular/router": "^5.2.0",
+    "@atlasmap/atlasmap.data.mapper": "1.33.0-SNAPSHOT.20180131184819",
```

Comment:
why it has changed to a "match" version from a "compatible with version"?

Reply:
Good question! It should actually be like that ("compatible", I mean) but the reason why the Angular upgrade took so long was precisely because the fragile stability originated by the massive amount of shared downstream dependencies between Syndesis itself and its own dependency tree (ngx-bootstrap, etc).

This forced us to track down each and every dependency and finetune its versioning to the path segment to ensure that it would play nicely not only with syndesis but with the rest of dependencies as well. In an ideal case scenario, NG dependencies will be distributed in a packaged artifact that implements the metadata info that Yarn requires, but it's not always like that, and those imported packages whose metadata is not complete or too open, might cause a class in other dependency's own dependencies graph. 

This PR aims to achieve a stable Angular 5 upgrade and then we'll go reviewing each dependency status on a per case basis.